### ISSUE TITLE
Example are failed, such as ShardingSpringBootMybatisExample.

### ISSUE THREAD

**Issue by zjcnb** [apache/shardingsphere#14916]
`master`

```
Exception in thread "main" java.lang.IllegalArgumentException: Cannot instantiate interface org.springframework.context.ApplicationContextInitializer : org.springframework.boot.context.ConfigurationWarningsApplicationContextInitializer
	at org.springframework.boot.SpringApplication.createSpringFactoriesInstances(SpringApplication.java:439)
	at org.springframework.boot.SpringApplication.getSpringFactoriesInstances(SpringApplication.java:418)
	at org.springframework.boot.SpringApplication.getSpringFactoriesInstances(SpringApplication.java:409)
	at org.springframework.boot.SpringApplication.<init>(SpringApplication.java:259)
	at org.springframework.boot.SpringApplication.<init>(SpringApplication.java:240)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1242)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1230)
	at org.apache.shardingsphere.example.sharding.spring.boot.mybatis.ShardingSpringBootMybatisExample.main(ShardingSpringBootMybatisExample.java:37)
Caused by: java.lang.NoSuchMethodError: org.springframework.core.KotlinDetector.isKotlinReflectPresent()Z
	at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:191)
	at org.springframework.boot.SpringApplication.createSpringFactoriesInstances(SpringApplication.java:435)
	... 7 more
```

**Comment by zjcnb** [comment]
@totalo Please help with this issue.

**Comment by totalo** [comment]
> @totalo Please help reslove this issue, thx.

ok, i will check.

**Comment by totalo** [comment]
The error is caused by the recent api adjustment but the sample code has not been adjusted. After discussion, we will put the generated source code into the generated folder, and then deal with it uniformly.

**Comment by zjcnb** [comment]
Fixed, refer to : #15404
